src/browser/index.jsx

```diff
@@ -150,6 +150,12 @@ var MainPage = React.createClass({
   handleOnTeamFocused: function(index) {
     // Turn off the flag to indicate whether unread message of active channel contains at current tab.
     this.markReadAtActive(index);
+
+    // TODO handle titles when switching teams correctly
+    ipcRenderer.send('update-title', {
+      title: remote.app.getName()
```

Comment:
Probably you can use `webview.getTitle()` by using index to get the element here.